server/utils/licenceStatus.js

```diff
@@ -194,13 +197,30 @@ function getExclusionTaskState(licence) {
 }
 
 function getCrdTimeState(licence) {
-
-    const timeAnswer = getIn(licence, ['eligibility', 'crdTime', 'decision']);
+    const decision = getIn(licence, ['eligibility', 'crdTime', 'decision']);
+    const dmApproval = getIn(licence, ['eligibility', 'crdTime', 'dmApproval']);
 
     return {
-        insufficientTime: timeAnswer === 'Yes',
-        crdTime: timeAnswer ? taskStates.DONE : taskStates.UNSTARTED
+        insufficientTimeContinue: dmApproval === 'Yes',
+        insufficientTime: decision === 'Yes',
```

Comment:
combine the above two to create ` insufficientTimeStop`, which would be true if insufficientTime=yes && insufficientTimeContinue == no